content/docs/actors/publishing.md

```diff
@@ -1,28 +1,29 @@
 ---
 title: Publishing your actor
 description: Prepare your actor for the Apify Store with a description and README file and learn how to make your actor available to the public.
-menuWeight: 4
+menuWeight: 7.5
 paths:
     - apify-platform/publishing-actors-on-apify-store/publishing-your-actor
+    - actors/publishing
```

Comment:
I'm not sure, but I think the old path should be removed. It can't redirect from docs.apify.com to developers.apify.com/academy AFAIK

Reply:
Fixed